components/ui/menuBoard/MenuBoard.tsx

```
@@ -0,0 +1,30 @@
+interface MenuBoardProps {
+  /** 표시할 메뉴 목록 */
+  menus: MenuInfo[];
+}
+
+/**
+ * 메뉴판 컴포넌트
+ * @param menus - 메뉴 이름과 가격 배열
+ */
+const MenuBoard = ({ menus }: MenuBoardProps) => {
+  return (
+    <ul className="flex w-full flex-col text-primary-text" role="list" aria-label="메뉴판">
```

Biome (2.4.6)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

(lint/a11y/useSemanticElements)